cdap-cli/src/main/java/co/cask/cdap/cli/CLIConfig.java

```diff
@@ -127,25 +127,28 @@ public TableRenderer getTableRenderer() {
   }
 
   public Id.Namespace getCurrentNamespace() {
-    return clientConfig.getNamespace();
+    return connectionConfig != null ? connectionConfig.getNamespace() : null;
```

Comment:
What does it mean for a CLIConfig to operate without a ConnectionConfig? Does it just mean that it will use the ClientConfig's internal ConnectionConfig?